src/styles/annotator/notebook.scss

```diff
@@ -24,4 +27,29 @@
   padding: 0;
   width: 100%;
   height: 100%;
+  border: none;
+}
+
+// This container element has the purpose of pushing children to the bottom-right end.
+.Notebook__controller-bar {
+  display: flex;
+  justify-content: flex-end;
+  background-color: var.$grey-2;
+}
+
+.Notebook__close-button {
+  position: absolute; // Comment this line to make the .Notebook__controller-bar to have height (> 0).
+  @include buttons.button--labeled(
+    $background-color: var.$grey-2,
+    $active-background-color: var.$grey-3
+  );
+  padding: var.$layout-space--xsmall;
+  font-size: var.$font-size--large;
+  margin-top: var.$layout-space--xxsmall;
+  margin-right: var.$layout-space--xxsmall;
+
+  // Center the button if icon-only
+  // .SvgIcon > svg {
+  // margin: 0;
+  // }
```

Comment:
If I'm understanding the intent here correctly, I'm guessing (but just guessing, could be wrong) that the `position:absolute` rule is intended to give the `.Notebook__controller-bar` no height? This causes a few weird side effects (and is a little hard to understand). Instead, might I suggest simplifying the rules here (and, while we're at it, handling spacing _around_ the button and font-sizing in the parent element instead of overriding button styles?

```suggestion
// This container element has the purpose of pushing children to the bottom-right end.
.Notebook__controller-bar {
  font-size: var.$font-size--large;
  display: flex;
  justify-content: flex-end;
  background-color: var.$grey-2;
  padding: 0.5em;
}

.Notebook__close-button {
  @include buttons.button--labeled(
    $background-color: var.$grey-2,
    $active-background-color: var.$grey-3
  );
```

Then, you can tighten up the vertical dead space on the top of the view by editing styles in `.HypothesisApp.scss`, something like this:

```
  &--notebook {
    padding: 0 var.$layout-space;
    @include responsive.respond-to(tablets desktops) {
      padding: 0 var.$layout-space--xlarge;
    }
  }
}
```

(In this case, zeroing out the top/bottom padding).

After, narrow screen:

![image](https://user-images.githubusercontent.com/439947/108898569-58a37500-75e5-11eb-8fd2-eca2f17ccd6d.png)

After, wider screen:

![image](https://user-images.githubusercontent.com/439947/108898602-62c57380-75e5-11eb-97d4-b85dde2a066a.png)

Reply:
The problem of removing the top/bottom padding for the `.HypothesisApp--notebook` is that it also affects the notebook when displayed standalone, leaving no vertical space on top or bottom:

![image](https://user-images.githubusercontent.com/8555781/109013297-41cb5400-76b3-11eb-902e-cf371e13b8d2.png)

Reply:
Alternatively we could remove the bottom padding from `.Notebook__controller-bar` and a little bit from top paddings from `.HypothesiApp--notebook`, as a compromise. This way it would still looks acceptable on both as standalone and within `<hypothesis-notebook>`.

Standalone | Within `<hypothesis-notebook>`
------------|----------------------------------|
![image](https://user-images.githubusercontent.com/8555781/109015545-835cfe80-76b5-11eb-87ba-b99b6a468560.png) | ![image](https://user-images.githubusercontent.com/8555781/109015958-f5cdde80-76b5-11eb-838a-524134acc87d.png)
 ![image](https://user-images.githubusercontent.com/8555781/109015681-b0a9ac80-76b5-11eb-8870-b4a412ee5301.png) | ![image](https://user-images.githubusercontent.com/8555781/109015872-dd5dc400-76b5-11eb-966b-f9d41180d011.png)